### DISCUSSION TITLE
Request for some control possibility of entinites

### DISCUSSION THREAD

**Discussion by Stormrage-DJ** [marcokreeft87/room-card#47] [Ideas]
Hi!

First of all I really like the already highly customize-able room card! I would like to request if possible some more additional control.
1:
- It would be nice to keep 1 entity on left and then align the rest to right (maybe as a possibility of a more sophisticated "alignment system"
- It would be nice if optionally entities without "state" could be aligned to match line with those who has

2:
- it would be great if "on demand" the icons to entities could be displayed, similar as shown on the left side (minimalistic-area-card)
![image](https://user-images.githubusercontent.com/83466158/190344344-08ba4e49-3368-40c1-bcab-a673ceff9404.png)

I'm looking forward and keeping on eye on the room-card development! Great work! Thanks!

**Comment by marcokreeft87** [comment]
Hey @Stormrage-DJ glad you like it! I dont really understand what you mean haha. Can you try to explain in an other way?

**Reply by Stormrage-DJ** [reply]
I've just noticed bullet points flow together, corrected in the original post.

So my goal would be in the case marked with "1" That I have a master entity
- for me it is the ceiling light, and i would like to put the "color, low, med, high" entity buttons to right side of the area, similar as you can see on the left side (that is made with multiple-entity-row)
- secondly I would like to align the "Ceiling Lights" and "Color, low, med high" in the same line, like if they would have "show_state" too.

**Comment by marcokreeft87** [comment]
The master entity would be the entity in config, so not the entities or info_entites.
The alignment would not be an issue if you disable the show_state for ceiling lights. Because the icon of ceiling ligt would show the state by being grey ( off ) or yellow ( on )
